### DISCUSSION TITLE
Database ingestion (MySQL, Postgres, elasticsearch)

### DISCUSSION THREAD

**Discussion by nimaforoughi** [cables-gl/cables_docs#702] [Answers & Questions]
Wondering if the databases can be directly queried by Cables and visualize data?
MySQL, Postgres and elasticsearch are the main ones I am looking for.
Also data massage is possible after data ingestion?

**Comment by steam0r** [comment]
you should be able to talk to any database that provides some HTTP api, elastic definitely does
